docs/source/cpp/compute.rst

```diff
@@ -682,17 +682,23 @@ String joining
 
 This function does the inverse of string splitting.
 
-+-----------------+-----------+----------------------+----------------+-------------------+---------+
-| Function name   | Arity     | Input type 1         | Input type 2   | Output type       | Notes   |
-+=================+===========+======================+================+===================+=========+
-| binary_join     | Binary    | List of string-like  | String-like    | String-like       | \(1)    |
-+-----------------+-----------+----------------------+----------------+-------------------+---------+
++-----------------+-----------+----------------------+----------------+-------------------+-----------------------+---------+
+| Function name   | Arity     | Input type 1         | Input type 2   | Output type       | Options class         | Notes   |
++=================+===========+======================+================+===================+=======================+=========+
+| binary_join     | Binary    | List of string-like  | String-like    | String-like       |                       | \(1)    |
++-----------------+-----------+----------------------+----------------+-------------------+-----------------------+---------+
+| var_args_join   | Varargs   | List of string-like  | (NA)           | String-like       | :struct:`JoinOptions` | \(2)    |
```

Comment:
For purposes of `var_args_join`, I interpret  "Input type 1" mean "the type of the 1st through n-1st argument" and "Input type 2" to mean "the type of the nth argument". So I think the table should be like this:
```suggestion
| var_args_join   | Varargs   | String-like          | String-like    | String-like       | :struct:`JoinOptions` | \(2)    |
```

Reply:
Thanks, that makes sense. I've updated the docs.